sqlness/src/runner.rs

```diff
@@ -59,9 +60,17 @@ impl<E: EnvController> Runner<E> {
             } else {
                 None
             };
-            let db = self.env_controller.start(&env, config_path).await;
-            let run_result = self.run_env(&env, &db).await;
-            self.env_controller.stop(&env, db).await;
+            let parallelism = self.config.parallelism.max(1);
```

Comment:
`min(1)`?

Reply:
`min` will become either 0 or 1. This is "take the min/max among two" (really misleading, I've fallen into this many times...)